src/v2/view-builder/views/IdentifierView.js

```diff
@@ -28,7 +29,26 @@ const Body = BaseForm.extend({
         }
       }), '.o-form-fieldset-container', false, true);
     }
-  }
+
+    this.listenTo(this.options.appState, 'formRendered', () => {
+      //add idps
+      if (this.options.appState.get('idps')) {
+        this.add(View.extend({
```

Comment:
I would recommend to create a new view for idp button list and then add it here instead of putting whole thing in this.add().